packages/asc-ui/src/components/Header/HeaderLogoText.tsx

```diff
@@ -1,27 +1,27 @@
-import React from 'react'
+import React, {
+  AnchorHTMLAttributes,
+  FunctionComponent,
+  HTMLAttributes,
+} from 'react'
+import { StyledProps } from 'styled-components'
 import HeaderLogoTextStyle from './HeaderLogoTextStyle'
 import HeaderTitle from './HeaderTitle'
 
-export interface LogoProps
-  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+export interface LogoProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
   href: string
   tall?: boolean
   title?: string
 }
 
-interface HeaderLogoTextProps extends React.HTMLAttributes<HTMLHeadingElement> {
+interface HeaderLogoTextProps extends HTMLAttributes<HTMLHeadingElement> {
   homeLink: string
   tall?: boolean
-  logo?: React.FC<LogoProps>
+  logo?: FunctionComponent<LogoProps>
 }
 
-const HeaderLogoText: React.FC<HeaderLogoTextProps> = ({
-  title,
-  homeLink,
-  tall,
-  logo: LogoIcon,
-  ...otherProps
-}) => (
+const HeaderLogoText: FunctionComponent<
+  HeaderLogoTextProps & StyledProps<any>
```

Comment:
It doesn't look like `StyledProps` actually adds the `as` prop to the type definition, but rather seems to ignore it. Perhaps the `StyledComponent` type would be a better option here?